.github/workflows/workflow_actions.yml

```diff
@@ -18,38 +18,40 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
   radas:
     runs-on: ubuntu-22.04
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v3
 
       - name: Install Poetry
-        run: curl -sSL https://install.python-poetry.org | python - --version 1.6.1
+        run: curl -sSL https://install.python-poetry.org | python - --version 1.8.2
 
       - name: Set up Python 3.11
         uses: actions/setup-python@v4
         with:
           python-version: '3.11'
           cache: 'poetry'
+      
+      - name: Setup
+        run: poetry install
 
       - name: Cache radas results
         id: radas
         uses: actions/cache@v3
         with:
           path: radas
-          key: radas-${{ hashFiles('.github/workflows/make_radas_data.sh')}}
+          key: radas-${{ hashFiles('./radas_dir')}}
```

Comment:
`radas_dir` can't be the source of the hash. Now that radas is a poetry dependency and thus part of `poetry.lock` using that as a hash is probably ok